protobufs/livekit_analytics.proto

```diff
@@ -151,8 +155,9 @@ message AnalyticsEvent {
   string error = 20;
   RTPStats rtp_stats = 21;
   int32 video_layer = 22;
+  SIPCallInfo sip_call = 24;
```

Comment:
we also need to send analytics a copy of the trunks. this will allow us to keep historical records